tests/token/erc1155/ERC1155BaseSuite.py

```diff
@@ -0,0 +1,808 @@
+import pytest
+from signers import MockSigner
+from utils import assert_event_emitted
+
+from nile.utils import (
+    MAX_UINT256, ZERO_ADDRESS, INVALID_UINT256, TRUE, FALSE,
+    to_uint, add_uint, sub_uint, assert_revert, str_to_felt
+)
+
+signer = MockSigner(123456789987654321)
+
+#
+# Helpers
+#
+
+
+def to_uint_array(arr):
+    return list(map(to_uint, arr))
+
+
+def prepare_calldata(arr):
```

Comment:
I think a comment/comments to clarify the expected formats to receive and return would be helpful (it is a hard-to-read method).